includes/Core/Util/Requires_Javascript_Trait.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Trait Google\Site_Kit\Core\Util\Require_Javascript_Trait
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Trait to display no javascript fallback message.
+ *
+ * @since 1.1.2
+ * @access private
+ * @ignore
+ */
+Trait Require_Javascript_Trait {
+
+	/**
+	 * Display fallback message when Javascript is disabled
+	 *
+	 * @since 1.1.2
+	 *
+	 * @return string noscript HTML tag,
+	 */
+	public function get_noscript_html() {
+
+		$no_script = sprintf('
+			<noscript>
+				<div class="mdc-layout-grid googlesitekit-no-js">
+					<div class="mdc-layout-grid__inner">
+						<div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-12">
+							<h3 class="googlesitekit-heading-2 googlesitekit-no-js__title">
+								%s
+							</h3>
+						</div>
+					</div>
+				</div>
+			</noscript>',
+			esc_html__( 'The Site Kit by Google plugin requires JavaScript to be enabled in your browser.', 'google-site-kit' )
+		);
+		
+		return apply_filters( 'googlesitekit_noscript_html', $no_script );
```

Comment:
Please remove this filter as it is not mentioned in the AC or IB.

Reply:
that was one suggestion I gave to @Shavindra but noted it would be a bit overkill and unnecessary here.

Would you say that if there's any need for a filter like this it would typically be mentioned in the IB correct?

Reply:
Yes, usually. There should at least be a good reason for adding it. In this case, I don't think there's much value in making the output here extensible for other plugins/themes to customize.